### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by illiashcherbak**

### CODE REVIEW

**Review by YakovlewDan — CHANGES_REQUESTED** [review]
Ok, but it may be better

**Review by YakovlewDan — APPROVED** [review]
Good job

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,24 @@
...
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  const shortRentTerm = 3;
+  const longRentTerm = 7;
+
+  const total = rentPerDay * days;
```
Comment (solaryasha) [inline src/calculateRentalCost.js:38]:
```suggestion
  const totalCost = rentPerDay * days;
```

**Review by solaryasha — APPROVED** [review]
👍

**Pull request closed without merging**